aten/src/ATen/SparseTensorImpl.cpp

```diff
@@ -29,58 +29,21 @@ namespace {
 // This means that we allocate a [1,0] size indices tensor and a [0] size
 // values tensor for such an empty tensor.
 SparseTensorImpl::SparseTensorImpl(at::TensorTypeId type_id, at::ScalarType scalar_type)
-    : TensorImpl(type_id, scalar_type, false)
-    , size_{0}
+    : TensorImpl(
+        type_id,
+        scalar_type,
+        /* is_variable */ false,
```

Comment:
Soon we should add a struct for all of these booleans :)

Reply:
Actually... I think you should make the struct in this PR. `true, true, true` is too much.